roc/log.go

```diff
@@ -183,6 +187,23 @@ func loggerRoutine() {
 	}
 }
 
+func logWrite(level LogLevel, text ...interface{}) {
+	if level <= loggerLevel {
+		_, file, line, _ := runtime.Caller(0)
```

Comment:
It seems we're running circles here.

As I already wrote, this will return file and line that corresponds to `log.go`. In other words, file and line will always be `log:go` and `192`.

What we need instead is to obtain the file and line which correspond to the place from which `logWrite()` was invoked.